cmd/collectors/rest/rest.go

```diff
@@ -350,7 +350,7 @@ func (r *Rest) processEndPoints() uint64 {
 		}
 
 		if len(records) == 0 {
-			r.Logger.Warn().Str("ApiPath", endpoint.prop.Query).Msg("no " + endpoint.prop.Query + " instances on cluster")
+			r.Logger.Debug().Str("ApiPath", endpoint.prop.Query).Msg("no " + endpoint.prop.Query + " instances on cluster")
```

Comment:
`2023-04-19T16:32:11+05:30 WRN rest/rest.go:353 > no api/protocols/cifs/services instances on cluster ApiPath=api/protocols/cifs/services Poller=sar collector=Rest:SVM`

`endpoint.prop.Query` already been logged in `ApiPath`, we don't need it twice.